ncm-metaconfig/src/main/metaconfig/slurm/pan/dbd.pan

```diff
@@ -6,8 +6,8 @@ include 'metaconfig/slurm/schema';
 bind "/software/components/metaconfig/services/{/etc/slurm/slurmdbd.conf}/contents" = slurm_dbd_conf;
 
 prefix "/software/components/metaconfig/services/{/etc/slurm/slurmdbd.conf}";
-"owner" = "root";
-"group" = "root";
-"mode" = 0644;
+"owner" = "slurm";
+"group" = "slurm";
+"mode" = 0600;
```

Comment:
Is this backwards compatible?

Reply:
yes, the slurmdbd daemon already ran as the slurm user. They just made it more strict with this version: the config file has to be closed off (as it contains the mysql credentials to access the db).

Reply:
Makes sense, thanks!